portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/StructuredContentFilteringAndSorting.testcase

```diff
@@ -139,4 +139,19 @@ definition {
 		}
 	}
 
+	@priority = "4"
+	test CanInvokeAllStructuredContentCreated {
+		property portal.acceptance = "true";
+
+		task ("When returning only web content with version 1.0") {
+			var response = HeadlessWebcontentAPI.versionStructureContent(versionvalue = "1.0");
+		}
+
+		task ("Then the response body only includes itens with version 1.0") {
+			TestUtils.assertEquals(
+				actual = "${response}",
+				expected = "Web Content 1,Web Content 2,Web Content 3,Web Content 4,");
```

Comment:
The ',' at the end is the result of using `StringUtil.add` I couldn't remove it, I tried the methods:
1. StringUtil.substring
2. StringUtil.extractLast